optimum/exporters/utils.py

```diff
@@ -150,7 +150,6 @@ def _get_submodels_for_export_diffusion(
 
     text_encoder_2 = getattr(pipeline, "text_encoder_2", None)
     if text_encoder_2 is not None:
-        text_encoder_2.config.output_hidden_states = True
```

Comment:
won't this be causing an issue here https://github.com/huggingface/optimum/blob/5ece6e88156de6a0de848ad6c1af381c81cfbcc3/optimum/exporters/onnx/model_configs.py#L975 ?

Reply:
it actually worked without, the problem is that clip text model with projection doesn't output the hidden states anymore, I finally found where it's coming from and it's fixed now (gotta set it in model.text_model as well)

Reply:
fixed